modules/nf-core/cellpose/main.nf

```diff
@@ -45,8 +45,10 @@ process CELLPOSE {
     }
     def prefix = task.ext.prefix ?: "${meta.id}"
     def VERSION = "2.2.2" // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    def name = image.name
+    def base = name.lastIndexOf('.') != -1 ? name[0..name.lastIndexOf('.') - 1] : name
```

Comment:
There is no way to customize the output file name?

Reply:
Unfortunately it is not possible with the current CLI: https://cellpose.readthedocs.io/en/latest/cli.html

Reply:
Allright that's fine :p